src/ngircd/log.c

```diff
@@ -63,12 +63,12 @@ Log_Message(int Level, const char *msg)
  * Initialitze logging.
  * This function is called before the configuration file is read in.
  *
- * @param Daemon_Mode Set to true if ngIRCd is running as daemon.
+ * @param Syslog_Mode Set to true if ngIRCd is running as daemon.
```

Comment:
Probably rephrase the comment as well? The "running as daemon" doesn't much sense once the variable is renamed, what do you think?